docs/customization.md

```diff
@@ -49,7 +49,7 @@ But you can implement your own logic by setting a callback.
 Example:
 ```java
 builder.blockingMethodCallback(it -> {
-    new Error(it.toString()).printStackTrace();
+    new BlockingOperationError(it.toString()).printStackTrace();
```

Comment:
since `printStackTrace` is used here, let's keep `new Error`, to also demonstrate that there is no magic in `BlockingOperationError` and any error can be used